### ISSUE TITLE
Icon broken

### ISSUE THREAD

**Issue by MiguelSRE** [umanamente/n8n-nodes-globals#9]
Hello, 

I just installed this node and the icon is broken. I’m attaching a screenshot.

![imagen](https://github.com/user-attachments/assets/c01716ef-f141-4117-b645-0da1bdf67e36)
![imagen](https://github.com/user-attachments/assets/a8835ea1-c0ba-4539-89e5-56658fc3f7f8)
![imagen](https://github.com/user-attachments/assets/8fcc9b9f-84af-40a0-9c6a-5c93fdce47ab)
![imagen](https://github.com/user-attachments/assets/88038a57-b768-4ebe-9a86-5858925f0dbc)

Regards,

**Comment by umanamente** [comment]
Please report this issue on the [N8N forum](https://community.n8n.io/c/questions/12) first, as it seems like the problem is with your system, not the node. Other users are seeing the icon displayed correctly, and no one else has reported this issue. 
If N8N support confirms that the problem is with the node, please reopen this issue and include a link to the support response.

**Comment by azdolinski** [comment]
@umanamente 

I have same problem.  
Problem is that in your NPM pacage - when you look on icon.. it has 9kb...
when you look on NPM 1.0.1 version:

![image](https://github.com/user-attachments/assets/76ef3c04-d7e8-4108-a945-057a7e1a3894)

in 1.0.0 looks to be ok..
![image](https://github.com/user-attachments/assets/78f69c56-60f2-425c-ac0a-f0768040a4f9)


when i upload original png from git - it looks to working so problem is inside npm only...

```bash
root@n8n:/opt/docker/n8n/data/nodes/node_modules/n8n-nodes-globals/dist/nodes/GlobalConstants# file --mime-type -b globals-icon-60px.png
image/png
root@n8n:/opt/docker/n8n/data/nodes/node_modules/n8n-nodes-globals/dist/nodes/GlobalConstants# file --mime-type -b globals-icon-60px.png_orig
application/octet-stream
```

**Comment by umanamente** [comment]
Fixed in v1.0.3.
Thank you for the details! The issue was caused by Gulp version 5.0.0, which produced corrupted images. Downgrading it resolved the problem.

**Comment by alejandrodizor** [comment]
Hey @umanamente Thanks for the great work with this node; it's been very useful to me. The icon displays perfectly on the node, but it doesn't show up in the credentials. Do you know if it's an error on my end or what could be causing this? Thanks in advance.

![image](https://github.com/user-attachments/assets/b1b898a5-5636-42d9-a098-b9782d663328)
